index.html

```diff
@@ -12,10 +12,16 @@
 <nav class="mainMenu">
     <p class ="menuLogo">Ukiyo</p>      
     
-    <div class="menuIcon"></div>
-    <div class="menuLvl"></div>
-    <div class="menuLvl"></div>
-    <div class="menuLvl"></div>        
+    <div class="menuIcon">
+        <div class="menuLvl"></div>
+        <div class="menuLvl"></div>
+        <div class="menuLvl"></div>        
+    </div>
+    <ul class="menuText">
+        <li><a>Portfolio</a></li>
+        <li><a>About</a></li>
+        <li><a>Contact</a></li>
+    </ul>
 </nav>
 
 <section class="homeContent">
```

Comment:
I have seen the following things that need to be redesigned according to the Desktop view.
1. `font-size` of all elements
2. Alignment of all elements according to Desktop view
3. Should take the Complete width and height according to Desktop View.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/221363048-7fc1f9cc-af83-4813-a742-4fc4553f78dc.png)| ![image](https://user-images.githubusercontent.com/61361037/221363099-4fed9a17-a348-40a3-a4a8-3d8b896b0920.png)|

Reply:
Hi @RaoAkif, thanks for your kind review! I just have a question: the activity asks to work with a 768px width screen, and as you can see on the attached files, it works just fine. What can I do so that it keeps the same design in a diffent screen proportion?

![01](https://user-images.githubusercontent.com/96145777/221370660-f8741f9e-dd54-4d38-b83f-b79ad8d964ec.jpg)
![02](https://user-images.githubusercontent.com/96145777/221370661-88039c87-3d30-457c-86af-33bf6b299d39.jpg)